ehr/resources/web/ehr/form/field/ProjectEntryField.js

```diff
@@ -344,11 +344,9 @@ Ext4.define('EHR.form.field.ProjectEntryField', {
         fromClient: true
       });
 
-      this.store.insert(0, newRec);
```

Comment:
Why dont you want to add this record to the store? If I read this right, you will end up setting the value but lack a record in the store to back it? Plus, isnt the point of creating the record here, as opposed to in UserEditableCombo, that the model has additional EHR-specific fields which get populated?